### DISCUSSION TITLE
Header for Wizard

### DISCUSSION THREAD

**Discussion by danielgsims** [cloudscape-design/components#1798] [General]
Hi there,

I am working on a page that uses a Wizard in the App Layout. Right now, the title of the page is the same as the title of the first step. I want to add more context to my users about the purpose of the Wizard. My only choice seems to be adding a long title for the first step. I'd like to be able to either change the heading of the Wizard for a given step, or add an overall heading to the Page. It looks like you could have done this with the Content Header slot but that's now deprecated. Does anyone have any advice?

**Comment by Al-Dani** [comment]
Hello Daniel,
Unfortunately, there is no option to add an extra header to the Wizard component. 

Wizard is designed for a creation flow, and should be triggered in a certain context, like clicking on a “Create” button, which means users should be aware of its general purpose.
In addition, page name in breadcrumb group conveys the context (see our [demo](https://cloudscape.design/examples/react/wizard.html)). 

We believe, it should be enough.

That said, could you share your use-case so we can give you more specific advice?
